routes/GET-post.js

```diff
@@ -27,11 +27,15 @@ module.exports = (app, db) => {
 			if (err) {
 				return res.redirect("/forum?tm=Internal Server Error 500&ts=false");
 			}
-
-			db.query(`SELECT email FROM cloudclub.logins WHERE id='${result[0].userid}'`, (err2, result2) => {
+			console.log("here")
+			db.query(`SELECT profile_path FROM cloudclub.profiles WHERE userid='${result[0].userid}' `, (err2, result2) => {
```

Comment:
Rename result2 variable name to something more meaningful like `profile_query_result`.